libr/debug/debug.c

```diff
@@ -32,11 +32,12 @@ static int r_debug_recoil(RDebug *dbg) {
 	dbg->reason.bpi = NULL;
 	if (ri) {
 		ut64 addr = r_reg_get_value (dbg->reg, ri);
-		recoil = r_bp_recoil (dbg->bp, addr);
+		recoil = r_bp_recoil (dbg->bp, addr - dbg->bpsize);
 		//eprintf ("[R2] Breakpoint recoil at 0x%"PFMT64x" = %d\n", addr, recoil);
 		if (recoil < 1)
 			recoil = 0; // XXX Hack :D
 		if (recoil) {
+			dbg->in_recoil = 1;
```

Comment:
It was already fixe din my commit

> On 15 May 2016, at 22:02, Joshua J. Drake <notifications@github.com> wrote:
> 
> In libr/debug/debug.c:
> 
> >  		//eprintf ("[R2] Breakpoint recoil at 0x%"PFMT64x" = %d\n", addr, recoil);
> >  		if (recoil < 1)
> >  			recoil = 0; // XXX Hack :D
> >  		if (recoil) {
> > +			dbg->in_recoil = 1;
> Fixed.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>